### PR TITLE
docs: add note about supported browsers

### PR DESCRIPTION
**Pull request by kvet**

### CODE REVIEW

File: packages/dx-react-grid/docs/guides/getting-started.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ This package does not contain visual components. In the examples below, visual c
 
   Make sure that [Material UI](https://material-ui-1dab0.firebaseapp.com/) dependencies are installed and properly configured. Check the Material UI's [Getting Started](https://material-ui-1dab0.firebaseapp.com/getting-started/installation) article for configuration details.
 
+### Supported Browsers
+
+React Grid supports the latest, stable releases of all major browsers: Chrome, Firefox, Safari, Opera, Microsoft Edge.
```
Comment (SergeyAlexeev) [inline packages/dx-react-grid/docs/guides/getting-started.md]: major -> popular? or just '..the latest version of the following browsers:...'

File: packages/dx-react-grid/docs/guides/getting-started.md [hunk 2]
```diff
@@ -32,6 +32,12 @@ This package does not contain visual components. In the examples below, visual c
...
+
+React Grid should operate well enough in other browsers which use the latest version of WebKit, Blink, or Gecko, whether directly or via the platform’s web view API. Though they are not officially supported. It is also applicable for Microsoft Interner Explorer. In order to operate correctly in some of these browsers, it may be required to add [polyfills](#polyfills). But some features may behave incorrectly due to a lack of required web API support.
```
Comment (SergeyAlexeev) [inline packages/dx-react-grid/docs/guides/getting-started.md]: `web API` sound like something from .net world )

File: packages/dx-react-grid/docs/guides/getting-started.md (hunk 1 above)
Comment (kvet) [reply packages/dx-react-grid/docs/guides/getting-started.md]: What is wrong with 'major browsers'?

Comment (SergeyAlexeev) [reply packages/dx-react-grid/docs/guides/getting-started.md]: Just sounds strange to me.

**Pull request merged**